### PR TITLE
[Admin Analytics] UI improvements

### PR DESCRIPTION
**Pull request by thenamankumar**
closes https://github.com/sourcegraph/sourcegraph/issues/41182
closes https://github.com/sourcegraph/sourcegraph/issues/41247
## Test plan

- Typos & copy update
- User management username column layout update

## App preview:

- [Web](https://sg-web-naman-admin-analytics-ui-imp.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-pyqsuioexf.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

File: client/web/src/site-admin/analytics/AnalyticsExtensionsPage/index.tsx [hunk 1]
```diff
@@ -192,7 +192,7 @@ export const AnalyticsExtensionsPage: React.FunctionComponent<RouteComponentProp
                                     ? `${groupingLabel} activity`
                                     : `${groupingLabel} unique users`
                             }
-                            labelX="Time"
+                            labelX="Date"
```
Comment (erzhtor) [inline client/web/src/site-admin/analytics/AnalyticsExtensionsPage/index.tsx:195]: I would suggest leaving this as "Time" to keep consistency with the rest of the analytics feature pages.

**Review by erzhtor — COMMENTED** [review]
Good job. Left one comment, and also, can we add the username to the user details dropdown, because in the case of a long username, it is being truncated, and there is no way to know more about it:
image

**Review by erzhtor — APPROVED** [review]

File: client/web/src/site-admin/analytics/AnalyticsExtensionsPage/index.tsx (hunk 1 above)
Comment (thenamankumar) [reply client/web/src/site-admin/analytics/AnalyticsExtensionsPage/index.tsx:195]: okay

Comment (thenamankumar) [reply client/web/src/site-admin/analytics/AnalyticsExtensionsPage/index.tsx:195]: done

Comment (thenamankumar) [reply client/web/src/site-admin/analytics/AnalyticsExtensionsPage/index.tsx]: Done

**Pull request merged**